_posts/2015-05-20-loosers-assemble.md

```diff
@@ -9,10 +9,22 @@ speakers:
  - Pete Graham - Github Export Tool (lightning talk)
  - James Booth - The difference between cows and horses (lightning talk)
  - Jed Richards - ES6/React personal project (lightning talk)
-
 ---
 
+## Jed ES6/React
+
+Repo is public in case anyone fancies having a poke around. Might add some "todo" issues soon, please get involved if you want! It's just a learning exercise. https://github.com/madebycomrades/autokitty-react
```

Comment:
Don't think Jekyll autolinks plain URLs in Markdown :disappointed: 

    [https://github.com/madebycomrades/autokitty-react](https://github.com/madebycomrades/autokitty-react)